definitions/json/CaseTypeTab.json

```diff
@@ -1003,27 +1029,27 @@
     "Channel": "CaseWorker",
     "TabID": "BroughtForwardDate",
     "TabLabel": "BF Actions",
-    "TabDisplayOrder": 13,
+    "TabDisplayOrder": 14,
     "CaseFieldID": "bfActions",
     "TabFieldDisplayOrder": 1,
     "DisplayContextParameter": "#TABLE(bfDate)"
   },
   {
     "CaseTypeID": "ET_EnglandWales",
     "Channel": "CaseWorker",
-    "TabID": "CaseHistory",
-    "TabLabel": "History",
-    "TabDisplayOrder": 14,
-    "CaseFieldID": "caseHistory",
+    "TabID": "CaseDocuments",
```

Comment:
History should be tab 15, Document should be tab 16. This was a change made to resolve an ExUI bug